packages/ncids-css/scss/components/nci-back-to-top/_back-to-top.scss

```
@@ -0,0 +1,73 @@
+.nci-footer__return-to-top {
```

we don't have a nci-footer, so probably should be:

```suggestion
.usa-footer__nci-return-to-top {
```